src/org/thoughtcrime/securesms/notifications/NotificationCenter.java

```diff
@@ -207,15 +206,21 @@ private String computeChannelId(String ledColor, boolean vibrate, @Nullable Uri
         return channelId;
     }
 
-    // return chatId from "ch_msgV_HASH.CHATID" or 0
-    private int parseNotificationChannelChatId(String channelId) {
+    // return Pair<accountId, chatId> from "ch_msgV_HASH.ACCOUNTID.CHATID" or Pair<0, 0>
```

Comment:
because the values are used and compared with zero in the existing code, also external classes pass zeros as method parameters, the Pair structure is handled only internally, could be reworked to use Pair/custom-class everywhere but I kept changes minimal

Reply:
added ChatData class and now return `null` here instead of (0, 0)